tests/testthat/test-post.R

```diff
@@ -24,3 +24,14 @@ test_that("can set alt text", {
   id <- upload_media_to_twitter(test_path("tweet.gif"), alt_text = "A bird tweeting")
   expect_type(id, "character")
 })
+
+
+test_that("post_tweet works", {
+  skip_if_offline()
+  skip_on_cran()
+  msg <- paste("test", Sys.time()) # To avoid having duplicated status
+  rt <- expect_message(post_tweet(msg), "your tweet has been posted!")
+  crt <- httr::content(rt)
+  rt <- post_destroy(as.character(crt$id))
+  expect_error(post_destroy(as.character(crt$id_str)))
```

Comment:
I'd suggest putting this in a separate test since it's no longer testing `post_tweet()`

Reply:
I don't understand, do you suggest to have it in a different file (test-post_destroy) or to split these 4 lines in two tests, one for post_tweet another for post_destroy? If I split the test I wouldn't be able to know which status/tweet I want to delete and then I would end up posting a new one and deleting it again which is what currently it is doing.

Reply:
You’d still deletion test the tweet in the creation test, but you’d only test the double deletion in the deletion. It is hard to keep creation and deletion tests disentangled in this sort of code but I think it’s worth striving for as much as possible. (I was just commenting on that one specific line, not the test in general)